### PR TITLE
May fix java.lang.NullPointerException

### PR DESCRIPTION
**Pull request by williampma**
Just added a basic "null" check at the line 140 mentioned in the email.  Not sure.

### CODE REVIEW

File: src/java/relex/output/OpenCogSchemeLink.java [hunk 1]
```diff
@@ -137,6 +137,9 @@ private String printDisjuncts()
 		public Boolean FNCallback(FeatureNode srcNode)
 		{
 			FeatureNode attr = srcNode.get("DISJUNCT");
+
```
Comment (misgeatgit) [inline src/java/relex/output/OpenCogSchemeLink.java]: seems you can join the two ifs with an or.

Comment (williampma) [inline src/java/relex/output/OpenCogSchemeLink.java]: Sure, may as well.

**Comment by AmeBel** [comment]
@williampma you can merge, you have access

**Comment by williampma** [comment]
OK, thx.

**Pull request merged**